plugin-infra/go-plugin-access/test/com/thoughtworks/go/plugin/access/analytics/AnalyticsExtensionTest.java

```diff
@@ -78,13 +90,23 @@ public void shouldTalkToPlugin_To_GetCapabilities() throws Exception {
     public void shouldTalkToPlugin_To_GetPipelineAnalytics() throws Exception {
         String responseBody = "{ \"view_path\": \"path/to/view\", \"data\": \"{}\" }";
         when(pluginManager.submitTo(eq(PLUGIN_ID), requestArgumentCaptor.capture())).thenReturn(new DefaultGoPluginApiResponse(SUCCESS_RESPONSE_CODE, responseBody));
+        AnalyticsPluginInfo pluginInfo = mock(AnalyticsPluginInfo.class);
```

Comment:
This seems like an unnecessary mock of a simple class. Can you consider this:

```java
AnalyticsPluginInfo pluginInfo = new AnalyticsPluginInfo(
    new GoPluginDescriptor(PLUGIN_ID, null, null, null, null, false),
    null, null, null);
    pluginInfo.setStaticAssetsPath("/assets/root");
    metadataStore.setPluginInfo(pluginInfo);
```

Ideally we should have a PluginInfoMother.